### ISSUE TITLE
[Proposal] createMany method for Models

### ISSUE THREAD

**Issue by manniL** [laravel/internals#852]
While we have a `saveMany` method for saving a lot of related models, there is no straightforward way to batch-create models on their own.

A `createMany` method (which also triggers events and touches created_at/updated_at) would be great IMO.

**Comment by xxRockOnxx** [comment]
I know there's already a `createMany` method, but it doesn't trigger events + touch the timestamps.

**Comment by manniL** [comment]
@xxRockOnxx Yup, that's the `Model::insert() ` method, which calls `Builder::insert()` under the hood. It's fine to use it, but as you said, no events are triggered and no timestamps are touched.

**Comment by xxRockOnxx** [comment]
I'm aware of its behavior but don't know why they made it that way. Would like to know also. Usually questions like this are supposed to be asked on IRC or slack AFAIK.

**Comment by chuck-wood** [comment]
Don't forget that `insert` doesn't do mass-assignment properly. (It disregards `$fillable` and `$guarded`.)
